test/helpers.ts

```diff
@@ -21,7 +21,8 @@ export async function execComposeDown(modules?: string[], region?: string) {
       logger.error(`Error cleaning db with error: ${e.message}`, e);
     }
   }
-  execSync('cd test && docker-compose down');
+  // Compose down and ensure everything gets tear down to avoid infinite test runs
+  execSync('cd test && docker-compose down --remove-orphans');
```

Comment:
I'm not convinced this will do that, since the issue (that I have seen) is a graphile worker still running and trying to talk to postgres. The test suite doesn't run inside of a docker container so that particular infinite loop would persist.